### DISCUSSION TITLE
Redis Authentication required error

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#13773] [Q&A] [answered]
Hi.
How can we add the Redis password on the sink block in the vector pipeline? 
I can't find any docs on this case

**Comment by jszwedko** [comment, marked as answer]
Hi @LinTechSo ! For simple username/password authentication you can provide the username and password in the connection URL:

```
redis://[<username>][:<password>@]<hostname>[:port][/<db>]
```
